packages/core/src/types.ts

```diff
@@ -2154,11 +2156,31 @@ export interface ActorSystemInfo {
 }
 
 export interface ActorSystem<T extends ActorSystemInfo> {
+  /**
+   * @private
+   */
```

Comment:
A neat thing that we could maybe utilize is `@internal` and TS `stripInternal: true` option